users/vsantillo/tap_dances/tap_dance_controller.h

```diff
@@ -0,0 +1,18 @@
+// Tap Dance Controller - returns a state based on the pattern pressed
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

```suggestion
#pragma once

// Tap Dance Controller - returns a state based on the pattern pressed
```